wordpress-mobile/WordPress-iOS
# Remove old Reader Detail and webview feature flag

This PR:

* Removes the old `ReaderDetailViewController`
* Removes the `readerWebview` feature flag.
* Renames `ReaderDeatilWebviewViewController` to `ReaderDetailViewController`
* Reorganize the Reader Detail files in the project

### To test

This PR doesn't add any new features. Just use the Reader (from the Reader tab, notifications, comments) and check that everything works fine. :)

PR submission checklist:
